src/makeCalculator.js

```diff
@@ -38,6 +38,35 @@
  */
 function makeCalculator() {
   // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+    add(a) {
+      calculator.result += a;
```

Comment:
use 'this' instead of 'calculator' inside the methods. Replace 'calculator.result' with 'this.result'. Apply for other cases as well

Reply:
With 'this.result' tests didn't work. I know how do that work (I saw this in fe_chat4) but I don't fully understand how this work. I must use calculator.call(this, number).